modules/exploits/windows/smb/ms17_010_eternalblue.rb

```diff
@@ -94,7 +94,8 @@ def initialize(info = {})
         'Notes' =>
           {
             'AKA' => ['ETERNALBLUE']
-          }
+          },
+        'DisclosureDate' => '2017-04-14'
```

Comment:
You have duplicated `DisclosureDate` with a new date (Shadow Brokers leak) instead of MS17-010's publication date.

Reply:
https://github.com/rapid7/metasploit-framework/pull/15182/files#diff-6c8fa9d1d6787fec553c948770aa3ac79ded0c6e2c346078953b8f3453ad35eeR152